client/login/wp-login/login-links.jsx

```diff
@@ -252,11 +252,7 @@ export class LoginLinks extends React.Component {
 			translate,
 			wccomFrom,
 			isGutenboarding,
-			locale,
 		} = this.props;
-
-		const langFragment = locale && locale !== 'en' ? `/${ locale }` : '';
```

Comment:
@ramonjd, looks like we need to reinstate this line now 😅

Reply:
And the above `locale` :)

Reply:
Ha. No worries. My bad for not properly testing. Thanks!

Reply:
What red squiggly line? :trollface: 

<img width="527" alt="Screen Shot 2020-04-07 at 10 30 26 am" src="https://user-images.githubusercontent.com/6458278/78617473-d02bc100-78ba-11ea-8252-de59aec741c7.png">